### PR TITLE
Add considerations for migration doc

### PR DESCRIPTION
**Pull request by caseyamp**
# Dev Docs PR

## Description

Add considerations for migration

## Deadline

Casual. 

## Change type

- [X] Doc update.

# PR checklist:

- [X] My documentation follows the style guidelines of this project.
- [X] I previewed my documentation on a local server using `mkdocs serve`.
- [X] Running `mkdocs serve` didn't generate any failures.
- [X] I have performed a self-review of my own documentation.

### CODE REVIEW

**Comment by caseyamp** [comment]
Fixes DDI-126

**Comment by caseyamp** [comment]
@justin-fiedler can you take a peek at this and LMK if it's what you had in mind?

File: docs/data/ampli/migration.md [hunk 1]
```diff
@@ -3,9 +3,21 @@ title: Migrate to Ampli
 description: If you already have analytics implemented in your product you can easily migrate.
 ---
 
-If you already have analytics implemented in your product and want to migrate, we make it easy to switch from your existing Analytics provider.
+If you already have analytics implemented in your product and want to migrate, Amplitude makes it easy to switch from your existing Analytics provider.
 
-### Migrating Events
+## Considerations
+
+- The `context` parameter isn't supported in Ampli. This was originally based on per Source Templates which is a feature that is eprecated. To add additional data on each event you can do so by creating a Middleware like below that will set "contextProp=true" on all events.
```
Comment (justin-fiedler) [inline docs/data/ampli/migration.md:10]: *deprecated

File: docs/data/ampli/migration.md [hunk 2]
```diff
@@ -23,4 +35,4 @@ ampli.songUploaded({
 });
 ```
 
-With all the benefits of type-safety, linting, and data validation to ensure that your analytics are accurate and trustworthy.
\ No newline at end of file
+With all the benefits of type-safety, linting, and data validation to make sure that your analytics are accurate and trustworthy.
```
Comment (justin-fiedler) [inline docs/data/ampli/migration.md:38]: Might read easier if we combing this with the blurb above:

> After you import your tracking plan into Amplitude Data, this event becomes the following, with all the benefits of type-safety, linting, and data validation to make sure that your analytics are accurate and trustworthy.

**Review by justin-fiedler — CHANGES_REQUESTED** [review]

**Review by justin-fiedler — APPROVED** [review]

**Pull request merged**